### ISSUE TITLE
AzureRM subnet cannot be created 

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-azurerm#104]
_This issue was originally opened by @c-yokoyama as hashicorp/terraform#15254. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

I tried to create virtual network and subnet  using AzureRM provider.
However,  `terraform apply` has not been finished and continued to output `Still creating`.

```
resource "azurerm_virtual_network" "vnet" {
  name                = "mynetwork"
  resource_group_name = "hoge"
  address_space       = ["10.0.0.0/8"]
  location            = "${var.zone}"
}

resource "azurerm_subnet" "subnet" {
  name                = "mynetwork"
  resource_group_name = "hoge"
  virtual_network_name = "${azurerm_virtual_network.vnet.name}"
  address_prefix       = "10.0.0.0/11"
}
```

I want to configure that vnet name and subnet name are the same. 
But, I think this is the course of this.

**Comment by tombuildsstuff** [comment]
Hi @c-yokoyama

Thanks for opening this issue - from what I can see, I believe this issue will be fixed by #6 - but I'll confirm this is fixed before merging.

Thanks!

**Comment by stack72** [comment]
Closed via #6
